docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,966 @@
+| **Version information**   | [`containerlab:0.42.0`](https://containerlab.dev/install/), [`srlinux:23.3.3`](https://github.com/nokia/srlinux-container-image),[`minikube v1.30.1`](https://minikube.sigs.k8s.io/docs/start/)|
+| **Authors**               | Míchel Redondo [:material-linkedin:][mr-linkedin]                                                                                                                                              |
+
+At the end of this blog post you can find a [quick summary](#tldr-version) of the steps.
+
+## Prerequisites
+
+The following tools are required to be installed to run the lab on any Linux host. The links will get you to the installation instructions.
+
+* The lab leverages [Containerlab](https://containerlab.dev/install/) to spin up a Leaf/Spine Fabric coupled with [Minikube](https://minikube.sigs.k8s.io/docs/start/) to deploy the Kubernetes cluster.
+
+* [Docker engine](https://docs.docker.com/engine/install/) has to be installed on the host system.
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/) CLI client is also required to interact with the k8s cluster.
+
+## Lab topology
+
+The goal of this lab is to provide users with an environment to test the network integration of a Kubernetes cluster with a Leaf/Spine SR Linux fabric.
+
+<figure markdown>
+  <div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph='{"page":0,"zoom":1.7,"highlight":"#0000ff","nav":true,"check-visible-state":true,"resize":true,"url":"https://raw.githubusercontent.com/srl-labs/srl-k8s-anycast-lab/main/images/topology.drawio"}'></div>
+  <figcaption>Topology</figcaption>
+</figure>
+
+The setup consists of:
+
+* A Leaf/Spine Fabric: 2xSpines, 4xLeaf switches
+* Minikube kubernetes cluster with MetalLB load balancing implementation (3 nodes)
+* Kubernetes service deployed on top of Cluster
+* Linux clients to simulate connections to k8s service (4 clients)
+
+Thanks to MetalLB, Kubernetes nodes establish BGP sessions with Leaf switches. In those sessions the IP addresses of the exposed services (loadBalancerIPs, commonly known as VIPs ) are to announced to the fabric.
+
+Clients will simulate connections to the VIP by using curl.
+
+## Kubernetes Service description
+
+The end service we will use on top of the kubernetes cluster is a Nginx HTTP echo server. This service will be deployed and exposed in all the k8s nodes. With simulated clients, we will verify how traffic is distributed among the different nodes/pods.
+
+## Underlay Networking
+
+The fabric is configured with [BGP unnumbered peering](https://documentation.nokia.com/srlinux/23-3/books/routing-protocols/bgp.html#bgp-unnumbered-peer). Spine switches act as BGP route reflectors for the BGP EVPN family.
```

Comment:
For underlay we need to add a diagram similar to this one 
where we show which ASN are assigned and who peers with who
https://app.diagrams.net/?src=about#Uhttps%3A%2F%2Fraw.githubusercontent.com%2Fsrl-labs%2Flearn-srlinux%2Fdiagrams%2Fquickstart.drawio#%7B%22pageId%22%3A%22TaE2ys9rEfVsJn4JcLC4%22%7D

Reply:
added